pr_check.sh

```diff
@@ -1,4 +1,28 @@
-set -exv
+#!/bin/bash
 
-pipenv install -d
-flake8
+# --------------------------------------------
+# Options that must be configured by app owner
+# --------------------------------------------
+export APP_NAME="ros"  # name of app-sre "application" folder this component lives in
+export COMPONENT_NAME="ros"  # name of app-sre "resourceTemplate" in deploy.yaml for this component
+export IMAGE="quay.io/repository/cloudservices/ros-backend"
+export DOCKERFILE="Dockerfile"
+export COMPONENTS_W_RESOURCES="vmaas"
+
+export IQE_PLUGINS="ros"
+export IQE_MARKER_EXPRESSION="ros_smoke"
+export IQE_FILTER_EXPRESSION=""
+export IQE_CJI_TIMEOUT="30m"
+
+
+# Install bonfire repo/initialize
+CICD_URL=https://raw.githubusercontent.com/RedHatInsights/bonfire/master/cicd
+curl -s $CICD_URL/bootstrap.sh > .cicd_bootstrap.sh && source .cicd_bootstrap.sh
+
+source $CICD_ROOT/build.sh
+
+# Deploy patchman to an ephemeral namespace for testing
```

Comment:
```suggestion
# Deploy ros to an ephemeral namespace for testing
```

Reply:
@mayurilahane I can still see `patchman` instead of `ros`. Can you please fix this?

Reply:
Hey my bad,
i committed local changes after that so it made this undo !
Thanks.